.github/config.yml

```
@@ -0,0 +1,29 @@
+# Comment to be posted on PRs from first-time contributors in your repository
+newPRWelcomeComment: >
+  🚀 **Awesome job, @{{user.login}}!** 🚀  
+  Thank you for submitting your first pull request! We love seeing new contributors.  
+  Be sure to review our contributing guidelines to ensure a smooth merge process.  
+  ![High Five](https://media.giphy.com/media/3oEjI6SIIHBdRxXI40/giphy.gif)
+
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+
+# Comment to be posted on pull requests merged by a first-time user
+firstPRMergeComment: >
+  🥳 **Congratulations, @{{user.login}}!** 🥳  
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)